notebooks/patient_readmission/patient-readmission-trust-results.ipynb

```diff
@@ -135,25 +140,25 @@
        "    <tr style=\"text-align: right;\">\n",
        "      <th></th>\n",
        "      <th>context</th>\n",
-       "      <th>type</th>\n",
-       "      <th>overall fairness</th>\n",
+       "      <th>type in []</th>\n",
```

Comment:
The column's name changed from `type` (e.g. burden) to `type in []`. Something seems wrong here because that column name is off and there are now multiple columns named `type in Feature (age)` that all have the value of "burden"...

Also, the column count jumped from 24 to 42 with this re-run (because of all the new `type in Feature (age)` columns), so columns like `Group details (AfricanAmerican)` aren't displayed.

Reply:
@laguirre-cs For (1), I'd forgotten about those undesirable side effects in construct_scores_dataframe. It now really doesnt work great for fairness. I couldnt find a clean way to fix it without lots of special casing. Anyway, I changed the example to use the fairness-specific functions. 

For (2), fixed